packages/peregrine/lib/talons/SearchBar/__tests__/useAutocomplete.spec.js

```diff
@@ -19,6 +19,11 @@ jest.mock('@apollo/react-hooks', () => {
     return { runQuery, queryResult, useLazyQuery };
 });
 
+// Could not figure out fakeTimers. Just mock debounce and call callback.
```

Comment:
Lodash debounce hates fakeTimers. We can switch to a smaller, simpler implementation, since we don't need all the Lodash options.

Reply:
Is `lodash.debounce` that big? I don't mind writing a smaller implementation but we already `_` as a dependency to the project.

Reply:
Lodash is surprisingly big in places. I don't think the debounce is big, necessarily, but it is overly complex.